tests/modules/helloscripting.c

```diff
@@ -28,6 +29,15 @@
  * CONSTI 432    # pushes the value 432 to the top of the stack
  * RETURN        # returns the current value on the top of the stack and marks
  *               # the end of the function declaration.
+ *
+ * FUNCTION sleep # declaration of function 'sleep'
```

Comment:
```suggestion
 * FUNCTION baz   # declaration of function 'baz'
```

To avoid overloading the name with the sleep instruction. The other functions here are foo and bar.